static/app/views/starfish/views/webServiceView/starfishView.tsx

```diff
@@ -212,7 +211,7 @@ export function StarfishView(props: BasePerformanceViewProps) {
             <MiniChartPanel title={t('Throughput Per Second')}>
               {renderThroughputChart()}
             </MiniChartPanel>
-            <MiniChartPanel title={t('Errors (5XXs)')}>
+            <MiniChartPanel title={t('5XX Responses')}>
```

Comment:
You can use the shared title here
```suggestion
            <MiniChartPanel title={DataTitles.errorCount}>
```

Reply:
I will do it in a follow up, don't want to run CI again 😅